osbenchmark/benchmark.py

```diff
@@ -581,9 +582,15 @@ def add_workload_source(subparser):
         action="store_true")
     test_execution_parser.add_argument(
         "--enable-worker-coordinator-profiling",
-        help="Enables a profiler for analyzing the performance of calls in Benchmark's worker coordinator (default: false).",
+        help="Enables a profiler for analyzing the performance of calls in OSB's worker coordinator (default: false). "
+            "Outputs to ~/.benchmark/logs/ as profile.log",
```

Comment:
Enables the Async Profiler to help analyze the performance...
The profiler output is saved in `~/.benchmark/logs/profile.log`.